package com.mm.exception_handling.try_catch;

public class ExceptionWithoutTryCatch {

	public static void main(String[] args) {
		System.out.println("Execution started of main() method");
		System.out.println(10/0);
		System.out.println("Execution completed of main() method");
	}
}
